src/lib/Scenes/Show/Components/ShowHeader/ShowHeader.tsx

```diff
@@ -185,17 +178,17 @@ export class ShowHeader extends React.Component<Props, State> {
             onItemSelected={this.handleArtistSelected.bind(this)}
             onViewAllPressed={this.handleViewAllArtistsPressed.bind(this)}
           />
-          <ButtonWrapper>
-            <Spacer m={1} mt={1} />
-            <InvertedButton
-              inProgress={isFollowedSaving}
-              text={is_followed ? "Show saved" : "Save show"}
-              selected={is_followed}
-              onPress={this.handleFollowShow}
-              grayBorder={true}
-            />
-            <Spacer m={1} />
-          </ButtonWrapper>
+          <Spacer mb={2} />
+          <Button
+            width="100%"
```

Comment:
I don't think percentage values work on RN. Is this being converted somehow to a flex value or something else?

Reply:
Our new Palette `<Button>` supports width as a prop as long as `block` is also passed and I was told to pass it this string with percentage, but I'll take a look at using a plain number value instead!

Reply:
I checked, looks like styled-components is changing percentage values to flex values.